src/components/Nav/NavMore.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './NavMore.scss';
+
+const NavMore = ({ setIsActive }) => {
+  return (
+    <div
+      className="moreCategory"
+      onMouseLeave={() => {
+        setIsActive(false);
+      }}
```

Comment:
```suggestion
const switchOffMenu = () => {
        setIsActive(false);
      }

  return (
    <div
      className="moreCategory"
      onMouseLeave={switchOffMenu}
```

이벤트 함수는 최대한 분리해서 JSX 위에 함수로 만들어주시고,
함수를 참조하는 방법으로 수정 해 보세요

JSX에는 렌더되는 태그들이 주로 남아있게 해주시는게 가독성과 유지보수에 좋습니다.